src/qt/transactiontablemodel.cpp

```diff
@@ -609,6 +609,22 @@ QVariant TransactionTableModel::data(const QModelIndex &index, int role) const
         return QString::fromStdString(rec->hash.ToString());
     case TxHexRole:
         return priv->getTxHex(rec);
+    case TxPlainTextRole:
+        {
+            QString details;
+            QDateTime date = QDateTime::fromTime_t(static_cast<uint>(rec->time));
+
+            details.append(date.toString("M/d/yy HH:mm"));
+            details.append(" ");
+            details.append(formatTxStatus(rec));
+            details.append(" (");
+            details.append(walletModel->getAddressTableModel()->labelForAddress(QString::fromStdString(rec->address)));
```

Comment:
nit: what about only append L620 to L622 if `!rec->address.empty()`? I guess most users have couple of transaction without according address labels.